### PR TITLE
Add readme linting

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,36 +64,38 @@ Note that when you set a custom path for a component, the global path prefix is
...
 
 ```scss
-@include oAssetsSetcomponentPaths((
-	'o-example': '/assets/example'
-));
+@include oAssetsSetModulePaths(
+	(
+		'o-example': '/assets/example'
+	)
+);
```
Comment (notlee) [inline README.md:77]: Is the indentation change here intentional?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 
 This component provides Sass and JavaScript utilities for reliably building paths to a component's static assets, such as CSS background images, fonts and JSON data files. This is needed because the URL path to load these assets will vary depending on how a product developer chooses to integrate the Origami component into their website.
 
 - [Overview](#overview)
-- [JavaScript](#javascript)
 - [Sass](#sass)
+- [JavaScript](#javascript)
 - [URL routing](#url-routing)
-- [Using with the build service](#using-with-the-build-service)
+	- [Using with the build service](#using-with-the-build-service)
```
Comment (notlee) [inline README.md:9]: lovely

File: README.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
-# Origami asset loader [![NPM version](https://img.shields.io/npm/v/@financial-times/health-check.svg)](https://www.npmjs.com/package/@financial-times/health-check) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
-
+# o-assets [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
```
Comment (notlee) [inline README.md:1]: 🙌

File: README.md (hunk 1 above)
Comment (chee) [reply README.md:77]: hmm! i don't think so ! i'll fix that

File: README.md (hunk 2 above)
Comment (chee) [reply README.md:9]: yes! it complains:

tooltip over this line of code showing: origami-component/readme-has-toc The table of contents items should be in the same order as the headings in the readme. Item "Using with the build service" does not match heading "Contact"

File: README.md (hunk 3 above)
Comment (chee) [reply README.md:1]: tooltip over this line of code showing: origami-component/readme-has-name The main heading in the README must be the component's name as defined in the bower.json. expected "o-assets", got "some asset loader"

:D

Comment (chee) [reply README.md:1]: tooltip over this line of code reading: origami-component/readme-has-name The main heading in the README must be the component's name as defined in the bower.json. expected "o-assets", got "Origami asset loader"
:D

File: .github/workflows/remark-lint.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  run:
+    name: run
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: set node.js 14.x
+      uses: actions/setup-node@master
+      with:
+        version: 14.x
```
Comment (JakeChampion) [inline .github/workflows/remark-lint.yml:12]:
```suggestion
        node-version: 14.x
```
I saw this from https://github.com/Financial-Times/o-assets/actions/runs/122695252
>Input 'version' has been deprecated with message: The version property will not be supported after October 1, 2019. Use node-version instead

Comment (chee) [reply .github/workflows/remark-lint.yml]: thank you:)

i've abandoned this for a while, and filed an issue on the preset repo: https://github.com/Financial-Times/remark-preset-lint-origami-component/issues/5

it was a rabbit hole, would rather continue the good work and look at it another time

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v3.4.7 :tada:

**Pull request merged**